Classes/common/CDTDatastore.m

```diff
@@ -39,6 +39,7 @@
 
 @interface CDTDatastore ()
 
+@property CDTDatastoreManager *manager;
```

Comment:
Will 'manager' change after the object is initialised? If not and although the property is private, I think we should make it `readonly`.